core/modules/wiki.js

```diff
@@ -179,7 +179,7 @@ exports.generateNewTitle = function(baseTitle,options) {
 		title = baseTitle;
 	while(this.tiddlerExists(title) || this.isShadowTiddler(title) || this.findDraft(title)) {
 		title = baseTitle + 
-			(options.prefix || " ") + 
+			(options.prefix === undefined ? " " : options.prefix) + 
```

Comment:
Why `=== undefined` and not `== null`? Wouldn't the later be more useful to capture both undefined and null values?

http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript